scripts/ag

```diff
@@ -61,16 +61,17 @@ def make(db, force):
     """Creates the specified database."""
     click.echo("Creating database")
     create_database(force)
-    build_and_initialize(verbose=True)
-    click.echo("Making settings table")
-    make_settings_table()
-    click.echo("Applying patches to database")
-    patch_db(verbose=True)
 
-    if db == 'test':
+    if db == 'production':
```

Comment:
Could you add a comment to the "test" branch that build/initialize and setttings table creation aren't necessary as they are already included in the test database?